package.json

```diff
@@ -27,6 +28,10 @@
     "scuttle-testbot": "^2.2.0",
     "ssb-ebt": "^9.1.2",
     "ssb-friends": "^5.1.7",
+    "ssb-friends-4": "npm:ssb-friends@^4.4.10",
+    "ssb-invite-2": "npm:ssb-invite@^2.1.7",
+    "ssb-conn-1": "npm:ssb-conn@^1.0.0",
```

Comment:
since we don't have a package-lock (and these are just dev deps) i think it's better to remove the `^`'s so we know we get the exact version used by ahau